src/components/Header/Header.jsx

```diff
@@ -9,12 +9,14 @@ class Header extends Component {
       <div className="header-container">
         <div className="left-container">
           <div className="jobhax-logo-container">
-            <Link to="/">
-              <div className="jobhax-logo" onClick={() => this.props.googleAuth.signOut()}></div>
+            <Link to="/dashboard">
+              <div className="jobhax-logo"></div>
             </Link>
           </div>
           <div className="search-box">
-            <img className="header-icon search-icon" src="../../src/assets/icons/SearchIcon@3x.png"></img>
+            <div className="header-icon search-icon">
+              <svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 24 24"><path className="svg_ics" d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/></svg>
```

Comment:
Our current icons are designed and drawn by our designer Yinan, we can request svg files for icons and then replace them. For now, its better to keep them as original png files.